### PR TITLE
Fix homework 2 instructions

### PR DESCRIPTION
**Pull request by jbouze**
The homework bot expects the string "\nHere is your description:" before the fortune telling sentence, just like in the video. This was missing in the homework instructions.
See also the discussion here: https://github.com/orgs/cpp-for-yourself/discussions/22

### CODE REVIEW

File: homeworks/homework_2/homework.md [hunk 1]
```diff
@@ -39,7 +39,8 @@ A C++ program that communicates with the user by using screams. It must:
     Please enter another adjective:
     ```
     And read it from the terminal as the user presses <kbd>Enter</kbd>
-7. Print the fortune telling that is based on the inputs that the user provided. For example:
+7. The homework bot will only mark the assignment as correct if the output matches **exactly** what it expects. Here, it expects a newline followed by the string "Here is your description:", just like in the video.
+8. Print the fortune telling that is based on the inputs that the user provided. For example:
```
Comment (niosus) [inline homeworks/homework_2/homework.md:43]: While I see that it is important to note that the output needs to match exactly, along with "Here is your description:" part, every point in this list iterates what the program expects and with this suggestion, point 7. will not really be such a suggestion. I would be happy if we rephrase it into something along the lines of the following instead:
> Print the phrase "Here is your description:" followed by a new line and a fortune telling generated by our code based on the inputs that the user provided. For example:
> ```
> Here is your description:
> Igor, the fearless STL guru that finds errors quicker than the compiler
> ```

How does that sound?

Comment (jbouze) [reply homeworks/homework_2/homework.md:43]: Sounds good, except that the new line come before the "Here is your description" part.
What is the correct way for me to submit this? Another commit on top of the previous one? Or from a new branch with just a single commit?

Comment (jbouze) [reply homeworks/homework_2/homework.md]: Answered my own question, squashed the commits.

Comment (niosus) [reply homeworks/homework_2/homework.md]: Sorry, this slipped through the cracks! Thanks for your correction!

**Pull request merged**